VERSION "1.0"


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:
PCM1 FOO
VAL_TABLE_ Table3 16 "16" 7 "7" 2 "2" 0 "0" ;
VAL_TABLE_ Table2  ;
VAL_TABLE_ Table1 1 "One" 0 "Zero" ;


BO_ 496 DashboardMessage: 8 Dashboard
 SG_ Temperature : 0|12@1- (1,-10) [-10|200] "degC"  PCM1,FOO
 SG_ Speed : 16|8@1+ (1,0) [0|250] "km/h" Vector__XXX
 SG_ DoorLock : 24|1@1+ (1,0) [0|0] "-" Vector__XXX
 SG_ SeatBelt : 32|1@1+ (1,0) [0|0] "-" Vector__XXX

BO_ 444 EmergencyMessage: 4 Vector__XXX
 SG_ CrashStat : 0|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ AirbagStat : 8|1@1+ (1,0) [0|0] "" Vector__XXX


CM_ BO_ 496 "Dashboard message used for Basic details of Vehicle.";
CM_ BO_ 444 "Emergency Message used for Information About Emergency Situation";


VAL_ 496 DoorLock 0 "Door_Open" 1 "Door_Lock" ;
VAL_ 444 CrashStat 1 "Crash_Detect" 0 "Vehicle_Safe";
VAL_ 444 AirbagStat 1 "Airbag_Released" 0 "Fine_Airbag";